spec/models/message_spec.rb

```diff
@@ -23,7 +24,7 @@
       it 'enqueues a job' do 
         expect {
           TestDatabaseMailer.with(email: contact.email).ping.deliver_later
-        }.to have_enqueued_job.on_queue('mailers')
+        }.to have_enqueued_job.on_queue('default')
```

Comment:
I think these should still be in the `mailers` queue.  Maybe there was a change to the way delayed messages get queued?

Reply:
I think so. It says here that the queue name has been changed.

https://github.com/rails/rails/blob/6-1-stable/actionmailer/CHANGELOG.md